starlark-test/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "starlark-test"
+edition = "2018"
+version = "0.0.0"
+authors = ["Damien Martin-Guillerez <dmarting@google.com>"]
```

Comment:
Can you take the opportunity to add yourself and Daniel here? (Also for the other crates)